localization/english/wc_buildings_l_english.yml

```
@@ -1,11 +1,55 @@
 ﻿l_english:
+ #Special Buildings
  building_type_dark_portal_01:0 "Dark Portal"
  building_type_dark_portal_01_desc:1 "A gateway that binds the two worlds of Azeroth and Draenor, allowing alien creatures to go back and forth."
  building_dark_portal_01:1 "Dark Rift"
  building_dark_portal_01_desc:1 "A tiny rift in the fabric of the universe that still connects the two worlds of Azeroth and Draenor."
  building_dark_portal_02:1 "Dark Portal"
  building_dark_portal_02_desc:1 "A massive arch with magical runes that maintains the connection between Azeroth and Draenor, allowing large armies to pass through the portal."
 
+ building_type_gurubashi_arena_01:0 "Gurubashi Arena"
+ building_type_gurubashi_arena_01_desc:1 "An arena for prisoners of war, kidnapped travelers, those in debt, and even some brave fools who fight for their freedom or for the spoils of victory."
+ building_gurubashi_arena_01:1 "Gurubashi Arena"
+ building_gurubashi_arena_01_desc:1 "The Gurubashi Arena provides ruffians a shot at competition, fame, and treasure. It is said that in ancient times, fights were to the death, and the sand of the area was muddy with troll blood."
+ 
+ building_type_karazhan_01:0 "Karazhan"
+ building_type_karazhan_01_desc:1 "A massive tower that watches over Deadwind Pass. Its inhabitants wield magical powers coveted by all of Azeroth."
+ building_karazhan_01:1 "Karazhan"
+ building_karazhan_01_desc:1 "Also known as "Medivh's Tower" or "Ivory Tower of Karazhan", Karazhan is an abandoned citadel located on a nexus of ley lines in southern Deadwind Pass. The tower is best known for its last known occupant Medivh, the last Guardian of Tirisfal."
+ 
+ building_type_lordaeron_palace_01:0 "Lordaeron Palace"
+ building_type_lordaeron_palace_01_desc:1 "A palace that holds together the domain of its ruler. It is a beacon of prosperity and unity to all who lay eyes upon it." 
+ building_lordaeron_palace_01:1 "Lordaeron Palace"
+ building_lordaeron_palace_01_desc:1 "The Palace of Lordaeron is the jewel of the Empire of Lordaeron and shows just how far humans have come. Its presence exudes an aura of prestige and with a hint of decadence."
+ 
+ building_type_temple_of_atal_hakkar_01:0 "Temple of Atal'Hakar"
```

```suggestion
 building_type_temple_of_atal_hakkar_01:0 "Temple of Atal'Hakkar"
```

I believe it's spelled [with two k's](https://wowpedia.fandom.com/wiki/Temple_of_Atal%27Hakkar)?